code/modules/surgery/bodyparts/_bodyparts.dm

```diff
@@ -784,42 +798,94 @@
 	drop_organs()
 	return ..()
 
-/**
- * Calculates how much blood this limb is losing per life tick
- *
- * Arguments:
- * * ignore_modifiers - If TRUE, ignore the bleed reduction for laying down and grabbing your limb
- */
-/obj/item/bodypart/proc/get_part_bleed_rate(ignore_modifiers = FALSE)
+/// INTERNAL PROC, DO NOT USE
+/// Properly sets us up to manage an inserted embeded object
+/obj/item/bodypart/proc/_embed_object(obj/item/embed)
+	if(embed in embedded_objects) // go away
```

Comment:
```suggestion
/// INTERNAL PROC, DO NOT USE
/// Properly sets us up to manage an inserted embeded object
/obj/item/bodypart/proc/_embed_object(obj/item/embed)
	PROTECTED_PROC(TRUE)
	if(embed in embedded_objects) // go away
```
if you dont want it used outside of internals, use the SMDMM lint for it ya dingus

Reply:
Good point

Reply:
actually, I don't think I can, since it's not called by a bodypart